docs/admin/config/site_config.mdx

```diff
@@ -201,7 +201,7 @@ All site configuration options and their default values are shown below.
 	// Disk usage threshold at which to display warning notification. Value is a percentage.
 	"gitserver.diskUsageWarningThreshold": 90,
 
-	// Configuration for logging and alerting, including to external services.
+	// Configuration for logging and alerting, including to external s.
```

Comment:
I don't know what "external s" means in this context. From the rest of this PR it _seems_ that this is in reference to Slack?